docs/content_management/collaborative_editing/install_collaborative_editing.md

```diff
@@ -0,0 +1,174 @@
+- session:
+    - `public_link_enabled` - determines whether the public link is available, default value: `false`, available values: `true`, `false`
+
+#### `ibexa\share` configuration
+
+To share content model, you need to configure the `ibexa\share` package.
+Under `ibexa.system` [configuration key](configuration.md#configuration-files), indicate the settings:
+
+``` yaml
+ibexa:
+    system:
+        admin_group:
+            share:
+                content_type_groups:
+                    - 'Content'
+                excluded_content_types:
+                    - 'tag'
+                    - 'product_category_tag'
+```
+
+The following setting is available:
+
+- `content_type_groups` – defines groups of content types for which the **Share** button is displayed (it can still be disabled for specific content types within these groups)
+
+In the example confugiration above, the **Share** button is displayed for any content that belongs to the `Content` group, except for `tag` and `product_category_tag` content types.
+
+### Add tables to the database
+
+Add the tables to the database:
+Create the `ibexa_share.sql` file that contains the following code:
+
+=== "MySQL"
+
+    ``` sql
+    [[= include_file('code_samples/collaboration/config/mysql/ibexa_share.sql', 0, None, '    ') =]]
+    ```
+
+=== "PostgreSQL"
+
+    ``` sql
+    [[= include_file('code_samples/collaboration/config/postgresql/ibexa_share.sql', 0, None, '    ') =]]
+    ```
+
+Then, run the following command, where `<database_name>` is the same name that you defined when you [installed](install_ibexa_dxp.md#change-installation-parameters) [[= product_name =]]:
+
+=== “MySQL”
+
+```bash
+mysql -u <username> -p <password> <database_name> < ibexa_share.sql
+```
+
+=== “PostgreSQL”
+
+```bash
+psql <database_name> < ibexa_share.sql
+```
+
+This command modifies the existing database schema by adding database configuration required for using Collaborative editing.
+
+### Modify the bundles file
+
+Then, if not using Symfony Flex, add the following code to the `config/bundles.php` file:
+
+``` php
+<?php
+
+return [
+    // A lot of bundles…
+    Ibexa\Bundle\Collaboration\IbexaCollaborationBundle::class => ['all' => true],
+    Ibexa\Bundle\Share\IbexaShareBundle::class => ['all' => true],
+    Ibexa\Bundle\FieldTypeRichTextRTE\IbexaFieldTypeRichTextRTEBundle::class => ['all' => true],
+    Ibexa\Bundle\CkeditorPremium\IbexaCkeditorPremiumBundle::class => [‘all’ => true],
+];
+```
+
+### Add migration file and execute migration
+
+Last step is to add migration file and execute migration with the following commands:
+
+``` bash
+php bin/console ibexa:migrations:import vendor/ibexa/collaboration/src/bundle/Resources/migrations/2025_08_26_10_14_shareable_user.yaml 
+php bin/console ibexa:migrations:migrate --file=2025_08_26_10_14_shareable_user.yaml
+```
```

Comment:
IMO this should be moved up, because it's not part of configuration. It's related to installation.